.changeset/nervous-pumpkins-matter.md

```diff
@@ -0,0 +1,36 @@
+---
+'@vanilla-extract/sprinkles': minor
+---
+
+Clean up public API, deprecating old API names. Also adding sprinkles to the docs site and using `sprinkles` in favour of `atoms` for the canoncial examples.
+
+API changes include:
+- Rename `createAtomicStyles` to `defineProperties`, `createAtomicStyles` is now deprecated
+- Rename `createAtomsFn` to `createSprinkles`, `createAtomsFn` is now deprecated
+- Rename `AtomicStyles` type to `SprinklesStyles`, `AtomicStyles` is now deprecated
```

Comment:
Should this type be `SprinklesProperties`?

Reply:
Yeah actually that sounds obvious now that you say it 👍